google-map-search.html

```diff
@@ -180,7 +186,7 @@
       }
 
       // Update location. This will trigger a new search.
-      this.location = new google.maps.LatLng(this.latitude, this.longitude);
+      this._setLocation(new google.maps.LatLng(this.latitude, this.longitude));
```

Comment:
is this calling the underlying setter that Polymer creates? Could you use `this.set('location', ...)` instead?